tardis/io/logger/logger.py

```diff
@@ -7,12 +7,11 @@
 
 warnings.filterwarnings("ignore", category=pyne.utils.QAWarning)
 
-FORMAT = "[$BOLD%(name)-20s$RESET][%(levelname)-18s]  %(message)s ($BOLD%(filename)s$RESET:%(lineno)d)"
+FORMAT = "[$BOLD{name:20s}$RESET][{levelname:18s}]  \n\t{message:s} ($BOLD{filename:s}$RESET:{lineno:d})"
```

Comment:
The format of the logging message has been changed from Python 2 to Python 3 `f-string`.

Reply:
See previous comment about the debug/non_debug variables

Reply:
Yes, I have made the required changes.